research_manager.py

```diff
@@ -1380,56 +1352,49 @@ def _generate_conversation_response(self, user_query: str) -> str:
             return f"I apologize, but I encountered an error processing your question: {str(e)}"
 
-    def get_multiline_conversation_input(self) -> str:
-        """Get multiline input with CTRL+D handling for conversation mode"""
-        buffer = []
-
-        # Save original terminal settings
-        fd = sys.stdin.fileno()
-        old_settings = termios.tcgetattr(fd)
-
-        try:
-            # Set terminal to raw mode
-            tty.setraw(fd)
-
-            current_line = []
-            while True:
-                char = sys.stdin.read(1)
-
-                # CTRL+D detection
-                if not char or ord(char) == 4:  # EOF or CTRL+D
-                    sys.stdout.write('\n')
-                    if current_line:
-                        buffer.append(''.join(current_line))
-                    return ' '.join(buffer).strip()
-
-                # Handle special characters
-                elif ord(char) == 13:  # Enter
-                    sys.stdout.write('\n')
-                    buffer.append(''.join(current_line))
-                    current_line = []
-
-                elif ord(char) == 127:  # Backspace
-                    if current_line:
-                        current_line.pop()
-                        sys.stdout.write('\b \b')
-
-                elif ord(char) == 3:  # CTRL+C
-                    sys.stdout.write('\n')
-                    return 'quit'
-
-                # Normal character
-                elif 32 <= ord(char) <= 126:  # Printable characters
-                    current_line.append(char)
-                    sys.stdout.write(char)
-
-                sys.stdout.flush()
-
-        finally:
-            # Restore terminal settings
-            termios.tcsetattr(fd, termios.TCSADRAIN, old_settings)
-            print()  # New line for clean display
-
+def get_multiline_conversation_input(self) -> str:
+  """Windows-compatible multiline input"""
+  buffer = []
+  current_line = []
+
+  try:
+      while True:
+          if msvcrt.kbhit():
+              char = msvcrt.getch()
+
+              # CTRL+D or CTRL+Z detection
+              if char in [b'\x04', b'\x1a']:
+                  sys.stdout.write('\n')
+                  if current_line:
+                      buffer.append(''.join(current_line))
+                  return ' '.join(buffer).strip()
+
+              # Handle special characters
+              elif char == b'\r':  # Enter
+                  sys.stdout.write('\n')
+                  buffer.append(''.join(current_line))
+                  current_line = []
+
+              elif char == b'\x08':  # Backspace
+                  if current_line:
+                      current_line.pop()
+                      sys.stdout.write('\b \b')
+
+              elif char == b'\x03':  # CTRL+C
+                  sys.stdout.write('\n')
+                  return 'quit'
+
+              # Normal character
+              elif 32 <= ord(char[0]) <= 126:
```

Comment:
should be "ord(char)", otherwise it always throws an exception and sends the quit signal, making the app exit conversation mode